src/amber/cli/templates/scaffold/view/src/views/{{name}}/show.ecr.ecr

```diff
@@ -4,7 +4,7 @@
 <p><%="<"%>%= <%= @name %>.<%= field.name %><%= field.reference? ? ".id" : "" %> %></p>
 <% end -%>
 <p>
-  <%="<"%>%= link_to("back", "/<%= Inflector.pluralize(@name) %>s", class: "btn btn-success btn-xs") -%>
```

Comment:
What are these changes about and how are they related to delete?

Reply:
Oh, ok Let'me remove this an open another PR :+1: